index.html

```diff
@@ -15,5 +15,89 @@
 
 <body>
 <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+<div class='container'>
+
+  
+  
+  <nav class="navigation">
+    <div class='aNav'>
+      <a class='a1' href='services.html'>Service</a>
+      <a href=''>Product</a>
+      <a href=''>Vision</a>
+      <a href=''>Features</a>
+      <a href=''>About</a>
+      <a href=''>Contact</a>
+    </div>
+    <div class='logoDiv'>
+      <img src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+  
+  </nav>
+
+  <header class ='innovationDiv'>
+      <div class='blockInn'>
+        <h1 class='innovation'>Innovation
+        On
+        Demand
+        </h1>
+      <button class='getStarted'>Get Started</button>
+      </div>
+      <div>
+      <img class=codePic src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+  </header>
+
+  <hr>
+
+  <section class="top-content">
+    <div class='featuresDiv'>
+      <h2 class='features'>Features</h2>
```

Comment:
It doesn't look like you're targeting this h2 class in your CSS, so you can leave the class off of it. Another thing to consider is that most of these H2 tags are identical in style on your page, so you could use the same class for all of them, and use specificity to target a particular element further